staging/src/github.com/kubeedge/beehive/pkg/core/core.go

```diff
@@ -42,7 +43,12 @@ func StartModules() {
 		beehiveContext.AddModule(&m)
 		beehiveContext.AddModuleGroup(name, module.module.Group())
 
-		go moduleKeeper(name, module, m)
+		if module.remote {
```

Comment:
Is the module restart determined based on the remote?  Sorry, I may need some time to get the definition of remote.